src/libs/Sound/BaseSound.ts

```diff
@@ -1,11 +1,17 @@
 import Onyx from 'react-native-onyx';
+import * as Browser from '@libs/Browser';
+import getPlatform from '@libs/getPlatform';
+import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 
 let isMuted = false;
 
 Onyx.connect({
-    key: ONYXKEYS.USER,
-    callback: (val) => (isMuted = !!val?.isMutedAllSounds),
+    key: ONYXKEYS.NVP_MUTED_PLATFORMS,
+    callback: (val) => {
+        const platform = Browser.isMobile() ? CONST.PLATFORM.MOBILEWEB : getPlatform();
```

Comment:
This isn't very DRY, should we maybe spin this out into a function (it will avoid problems in future too)?